addons/mail/static/src/new/web/chat/chat_window.js

```diff
@@ -1,24 +1,24 @@
 /* @odoo-module */
 
-import { Call } from "../rtc/call";
-import { Thread } from "../core_ui/thread";
-import { Composer } from "../composer/composer";
-import { useStore } from "../core/messaging_hook";
-import { useRtc } from "../rtc/rtc_hook";
+import { Call } from "../../rtc/call";
+import { Thread } from "../../core_ui/thread";
+import { Composer } from "../../composer/composer";
+import { useStore } from "../../core/messaging_hook";
+import { useRtc } from "../../rtc/rtc_hook";
 import { useMessageEdition, useMessageHighlight, useMessageToReplyTo } from "@mail/new/utils/hooks";
 import { Component, useChildSubEnv, useRef, useState } from "@odoo/owl";
 import { useService } from "@web/core/utils/hooks";
 import { localization } from "@web/core/l10n/localization";
-import { CallSettings } from "../rtc/call_settings";
-import { ChannelMemberList } from "../discuss/channel_member_list";
+import { CallSettings } from "../../rtc/call_settings";
+import { ChannelMemberList } from "../../discuss/channel_member_list";
 import { ChatWindowIcon } from "./chat_window_icon";
-import { ChannelInvitationForm } from "../discuss/channel_invitation_form";
-import { isEventHandled } from "../utils/misc";
+import { ChannelInvitationForm } from "../../discuss/channel_invitation_form";
+import { isEventHandled } from "../../utils/misc";
 import { ChannelSelector } from "@mail/new/discuss/channel_selector";
 
 /**
  * @typedef {Object} Props
- * @property {import("@mail/new/chat/chat_window_model").ChatWindow} chatWindow
+ * @property {import("@mail/new/web/chat/chat_window_model").ChatWindow} chatWindow
```

Comment:
`/chat/` folder is very confusing in our context, I would take the opportunity to rename to `/chat_window/`